tests/test_stats_bolt.py

```
@@ -0,0 +1,36 @@
+    data = response.json()
+    assert data["total_issues"] == 3
+    assert data["resolved_issues"] == 2
+    assert data["pending_issues"] == 1
+    assert data["issues_by_category"]["Road"] == 2
+    print("Stats Response:", data)
+
+if __name__ == "__main__":
+    try:
+        test_get_stats()
+        print("Stats verification successful!")
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)